siad/daemon.go

```diff
@@ -125,6 +139,7 @@ func startDaemon(config Config) (err error) {
 		if err != nil {
 			return err
 		}
+		defer g.Close()
```

Comment:
Yeah.  There's a separate type `Server` defined there, that does close the modules.  It's only used for testing now.